src/overlay_12_0226BEC4.c

```diff
@@ -0,0 +1,60 @@
+#include "overlay_12_0226BEC4.h"
+#include "battle_system.h"
+#include "overlay_12_02266024.h"
+#include "heap.h"
+#include "unk_0201F79C.h"
+#include "unk_02005D10.h"
+
+static void ov12_0226BF04(SysTask *task, void *data);
+
+SysTask *ov12_0226BEC4(BattleSystem *bsys) {
+    UnkBattleStruct_0226BEC4 *unk;
+    unk = AllocFromHeap(HEAP_ID_BATTLE, sizeof(UnkBattleStruct_0226BEC4));
+    MI_CpuFill8(unk, 0, sizeof(UnkBattleStruct_0226BEC4));
+    unk->bsys = bsys;
+    return CreateSysTask(ov12_0226BF04, unk, 1000);
+}
+
+void ov12_0226BEF0(SysTask *task) {
+    FreeToHeap(sub_0201F988(task));
+    DestroySysTask(task);
+}
+
+static void ov12_0226BF04(SysTask *task, void *_data) {
+    UnkBattleStruct_0226BEC4 *data = _data;
+    u32 *unkPtr = ov12_0223A900(data->bsys);
+    
+    switch (data->state) {
+    case 0:
+        GF_ASSERT(unkPtr != NULL);
+        {
+            NARC *bgHandleNarc = NARC_ctor(NARC_a_0_0_7, HEAP_ID_BATTLE);
```

Comment:
ctor -> New

Reply:
New or Open? Narcs are a lil different

Reply:
either or, just not ctor

Reply:
thinking about it, I'm actually unsure if it will, there will be two commits making the same changes and I'm not 100% sure how git will handle it, it may just decide that the second commit is redundant and allow it to merge, or it may complain

Reply:
I've just done some tests in a [dumb repository](https://github.com/Laquin-Laquih/Test0001/) and I had no issue merging identical PRs. The conflict appeared when the same line suffered different modifications in each PR (as in #150), but in this case we shouldn't have issues. Anyways, if @adrienntindall rebased the branch before merging the PR, then I think it's guaranteed that rest will be straightforward.